Client/Frontend/Settings/BraveShieldsAndPrivacySettingsController.swift

```diff
@@ -104,15 +104,21 @@ class BraveShieldsAndPrivacySettingsController: TableViewController {
         return shields
     }()
     
-    private lazy var toggles: [Bool] = {
```

Comment:
Not a huge deal, but now that this is no longer lazy, this will be executed for every clearable we have which seems a bit wasteful since it will evaluate to the exact same value each time.

Perhaps it would be easier to just set `Preferences.Privacy.clearPrivateDataToggles.value` in the [`valueChanged`  closure where `toggles[idx]` is set](https://github.com/brave/brave-ios/blob/3a040a87756f7dae5592714b2c0346c839f14da6/Client/Frontend/Settings/BraveShieldsAndPrivacySettingsController.swift#L146)?

Reply:
Purpose was to gather the logic related with toggles to handle by itself but you are right it is wasteful.

Fix at https://github.com/brave/brave-ios/pull/3960/commits/3a7dfb3304ce58a43ed43738c9d7c7097d89b610